src/specter_edn/core.clj

```diff
@@ -105,9 +108,19 @@
   [node coll children]
   ((inner-node-ctor node coll) children))
 
+(defn- has-whitespace?
+  [output-nodes new-output]
+  (or (empty? output-nodes)
+      (w/whitespace? (last output-nodes))
```

Comment:
`(n/printable-only? ...)` is a superset of `(n/whitespace? ...)`, so it's not the same.